Docs: Update UG formatting + Errors (#353)
AY2324S2-CS2103T-T17-3/tp: docs/UserGuide.md

>   * `tricep dips` added with default values.
> ![fitadd-arms](images/FitaddArms.png)
</div>

Examples:

* `fitadd 1 n/burpees` - Adds `burpees` exercise to the client at index `1` with the following values:
  * `Sets`: 1 (default value).
  * `Reps`: 1 (default value).
  * `Break between sets`: 0 seconds (default value).

  If `burpees` already exists, please refer to this [warning](#fitadd-overwrite-warning) for the behaviour of this command.

* `fitadd 1 n/burpees r/5` - Adds `burpees` exercise to the client at index `1` with the following values:
  * `Sets`: 1 (default value).
  * `Reps`: 5 (specified value).
  * `Break between sets`: 0 seconds (default value).

  If `burpees` already exists, please refer to this [warning](#fitadd-overwrite-warning) for the behaviour of this command.

* `fitadd 1 n/burpees s/3 r/5 b/30` - Adds `burpees` exercise to the client at index `1` with the following values:
  * `Sets`: 3 (specified value).
  * `Reps`: 5 (specified value).
  * `Break between sets`: 30 seconds (specified value).

  If `burpees` already exists, please refer to this [warning](#fitadd-overwrite-warning) for the behaviour of this command.

> 

Reviewer: The overwrite warning for each example seems a little repetitive.

Reviewer 2: I agree, would it be better to put it all under one sentence instead?

Author: Alright, made the necessary changes already. Please review it again :)

* If no prefix is specified to search for a client's name, it must be the first field to search.
    * e.g. `find t/friends roy` will result in an error.
* If name prefix is specified it will take precedence over the non-prefixed argument.
    * e.g. `find roys n/roy` will use `n/roy` to search for a client's name instead of `roys`.
</div>

Examples:

> * `find n/Wendy` - Finds all clients whose name contains `Wendy`
> * `find n/Wendy t/Lover` - Finds all clients who name contains `Wendy` and is tagged with `Lover`

Reviewer: ```suggestion
* `find n/Wendy` - Finds all clients whose names contain `Wendy`
* `find n/Wendy t/Lover` - Finds all clients whose names contain `Wendy` and are tagged with `Lover`
```
